docs/dxp/7.x/en/installation-and-upgrades/setting-up-liferay-dxp/clustering-for-high-availability/clustering-search.md

```diff
@@ -15,6 +15,8 @@ Once DXP servers and Elasticsearch have both been properly configured as a clust
 
     ![Change your Elasticsearch configuration to Remote mode from the Control Panel.](./clustering-search/images/01.png)
 
+<!-- no other configuration required other than choosing "Remote" mode? -->
+
```

Comment:
@jrhoun - I mentioned this article to Russ and I think we agreed that he'd revise it after it's merged.